src/components/ImageView/index.native.js

```diff
@@ -138,12 +139,6 @@ class ImageView extends PureComponent {
                     styles.justifyContentCenter,
                     styles.overflowHidden,
                 ]}
-                onLayout={(event) => {
-                    const layout = event.nativeEvent.layout;
-                    this.setState({
-                        containerHeight: layout.height,
-                    });
-                }}
```

Comment:
@JediWattson 

Why should we remove this one, seem these changes are added on PR https://github.com/Expensify/App/pull/12648 to fix https://github.com/Expensify/App/issues/10792

cc: @luacmartins @chiragsalian

Reply:
@Santhosh-Sellavel It appears that commit just moves this code into a different position in the file. I didn't look into it, but my belief is that this was used before `withWindowDimensions` was added and never got removed. However the state unloads in android while the component stays loaded and this causes the issue with the error.

Reply:
Yeah you are right it was just moved there, actually it was added here https://github.com/Expensify/App/pull/8804 to address this [issue](https://github.com/Expensify/App/issues/8115)

Reply:
It brings back that issue 

#### Attachment Add
![Screenshot_1675121158](https://user-images.githubusercontent.com/85645967/215619768-ccbda0c4-0c8f-4904-8af9-1ca8b2851b36.png)

#### Attachment View
![Screenshot_1675121232](https://user-images.githubusercontent.com/85645967/215619878-81d8ed40-9706-4583-a63b-2c9614fa0e62.png)

#### This is the actual image
![Screenshot_1675120950](https://user-images.githubusercontent.com/85645967/215619894-539ca4f3-98ba-4b51-8bea-0dabc9c14d10.png)

Reply:
I just pushed changes that should fix this!